docs/tutorial/test_model.rst

```diff
@@ -11,19 +11,32 @@ you will see the interface of the toolbox:
 
 .. image:: ../pictures/ModelTest1.jpg
 
-Use the Properties Window (1) to choose the RodPendulumModel, 
-the ODEInt Solver and the AdditiveMixer as ModelMixer. 
+Within the Properties Window (1), double clicking on a value (all :py:data:`None` by default)
+activates a drop down menu.
+Clicking again presents all eligible options.
+One of these options as an eligible model is the class RodPendulumModel,
+since it was registered and linked to PyMoskito earlier.
```

Comment:
Like above, maybe go for "One of these options now shows the :py:class:`RodPendulumModel` since it was introduced to the toolbox by registering it earlier.